### ISSUE TITLE
Remove Coinbase from the "Choose your Wallet" page 

### ISSUE THREAD

**Issue by laudaa** [bitcoin-dot-org/bitcoin.org#1254]
The CEO declared that Coinbase is not a wallet. Source:
https://medium.com/@barmstrong/coinbase-is-not-a-wallet-b5b9293ca0e7#.yr03e9u7y

There was a pull request in the past: https://github.com/bitcoin-dot-org/bitcoin.org/pull/1178

**Comment by ABISprotocol** [comment]
Coinbase claims as per that medium post cited by @laudaa that it is a "hybrid wallet/exchange" and that "Over the next year or so, you’ll see the Coinbase brand shift from being a hybrid wallet/exchange to focusing on purely being a retail and institutional exchange."  However, Coinbase is actually a brokerage. Please see issue #1109 and corresponding pull request #1223 for further discussion on related matters.

**Comment by crwatkins** [comment]
I fear that this going to sound a bit snarky, but I assure you that I mean it quite seriously:  There are no criteria in bitcoin.org's wallet listing requirements that a listing be "called" a wallet by any particular entity.  Further, in the source referenced, Brian Armstrong states:

> I also think a large number of people will continue to use Coinbase as a wallet (we have the #1 wallet on iOS and #2 on Android for example) and that is perfectly ok, even if it isn’t our focus.

I do not recommend removing the Coinbase listing.

**Comment by VincentPiper** [comment]
Coinbase under U.S. permitting, regulation, and law is considered to be a money transmitter. 
Laws and proposals are being drafted now to clarify the distinction between a device that stores a private key (aka a wallet) and an exchange that facilitates the buying and selling of bitcoin (aka money transmitter).

Coinbase's primary function is as an regulated exchange, and their wallet is only a secondary system to and in support of their use as that exchange. To be more clear, it is an "exchange wallet", the same way that an account that stores cash in between stock trades in a brokerage, is your "brokerage cash account", not your "bank savings account".

For those reasons, I believe Coinbase should be removed from the listing of approved wallets and an approved exchange section should be created, and they can be added there.

Wallets and Exchanges (with a wallet) are legally are two separate systems.

**Comment by laudaa** [comment]
An additional list could be created to add "exchange wallets" or similar services where their wallet is the secondary system. I don't think that new users should be mislead like this. This can also cause confusion (it being listed under wallets and CEO stating that it isn't a wallet).

**Comment by crwatkins** [comment]
My proposal is here https://github.com/bitcoin-dot-org/bitcoin.org/issues/1109#issuecomment-162143702

**Comment by SartoNess** [comment]
Dear Miss or Mr. Your issue subject it's wrong. I have the title you'd like to write but you did not. 
Remove Coinbase from the "Choose your Wallet" page  **because coinbase support Bitcoin Classic**
Now the issue:Coinbase have strong security,users can withdraw and deposit,therefore it can be used as a wallet, irrespective Brian Armstrong post.

**Comment by VincentPiper** [comment]
> Now the issue:Coinbase have strong security,users can withdraw and deposit,therefore it can be used as a wallet,

Well if that is the only criteria for determining what a web wallet is, I'd suggest you also add all other exchanges that also contain a wallet for the sake of consistency. For example, Gemini, Bitfinix, Bitstamp, btc-e, OKCoin, Huobi, Kraken, etc.

**Comment by HostFat** [comment]
Maybe we can say that it is a wallet until the user isn't forced to do the KYC just to use the wallet part, receiving/sending bitcoin. (so, without touching fiat money)

Does Coinbase force the user to give their private information just to receive some Bitcoin? (I can't remember)

The other possibility, is adding a sub-category for the services/wallet that force the user to give private informations.

**Comment by ABISprotocol** [comment]
Regarding Coinbase itself and what it might be called, I wish to emphasize as I did previously, that  per that medium post cited by @laudaa that it is a "hybrid wallet/exchange" and that "Over the next year or so, you’ll see the Coinbase brand shift from being a hybrid wallet/exchange to focusing on purely being a retail and institutional exchange." However, Coinbase is actually a brokerage (see earlier comment for basis for this claim).  As someone has also pointed out, it is regulated as a money transmitter.

A wallet (in particular, a wallet which is featured on bitcoin.org) should not serve the purpose of advancing the interest of a brokerage, bank or banking network (distributed ledger). Brokerages act as a middleman, buying bitcoin in bulk and selling them to customers. Brokerages charge a fee for this service. Exchanges, such as BitStamp, and BTC-E, offer direct buying and selling between currencies. Once funded,exchanges allow for quick buying and selling within the platform.
Trading fees may be low, but converting funds to and from bank accounts may be expensive and take considerable time. (Refer to: https://btcfoundationedcom.github.io/guides/pdf/buying_letter.pdf)

As I understand it, @crwatkins is not inclined to support the optional criteria pull request which I have submitted at #1223 - [he has proposed to create a single separate category with criteria listed](https://github.com/bitcoin-dot-org/bitcoin.org/issues/1109#issuecomment-162143702).  This could incorporate some of the changes I have proposed or be something else entirely.  I hope that @crwatkins or someone else will do a pull request that will fulfill this issue.
